Amplify/Core/Support/AmplifyAsyncThrowingSequence.swift

```diff
@@ -7,12 +7,15 @@
 
 import Foundation
 
-public struct AsyncThrowingChannel<Element: Sendable>: AsyncSequence {
+public struct AmplifyAsyncThrowingSequence<Element: Sendable>: AsyncSequence {
```

Comment:
```suggestion
public struct AmplifyAsyncThrowingSequence<Element: Sendable>: AsyncSequence, Cancellable {
```
Should conform to `Cancellable`